src/frontend/web_application/package.json

```diff
@@ -69,13 +69,15 @@
     "locale": "^0.1.0",
     "lodash.isequal": "^4.5.0",
     "lodash.throttle": "^4.1.1",
+    "moment": "^2.18.1",
```

Comment:
`@gandi/react-translate` includes `moment@2.17.1` and `react-datepicker` includes `moment@2.18.1`

might want to make `moment: >=2.17.1` a [dev/peer]Dep of `@gandi/react-translate`